### PR TITLE
Bugfix/AKDS-117/tablet sizes-rebased-with-dev/master

### PR DESCRIPTION
**Pull request by ksanmartin909**
**Includes:**
AKDS-105
https://filamentgames.atlassian.net/browse/AKDS-105

AKDS-117
https://filamentgames.atlassian.net/browse/AKDS-117

AKDS-111
https://filamentgames.atlassian.net/browse/AKDS-111

AKDS-119
https://filamentgames.atlassian.net/browse/AKDS-119

ADKS-118
https://filamentgames.atlassian.net/browse/AKDS-118

AKDS-115
https://filamentgames.atlassian.net/browse/AKDS-115

AKDS-110
https://filamentgames.atlassian.net/browse/AKDS-110

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -908,7 +907,10 @@ body,
   }
 }
 
-@media (max-width: 1170px) {
+
+
+/* iPad Air - Landscape */
+@media (max-width: 1180px) {
```
Comment (ksanmartin909) [inline src/App.css:913]: Adjusted this breakpoint up a bit to capture/target the iPad Air

File: src/App.css [hunk 2]
```diff
@@ -957,36 +959,32 @@ body,
...
   .earth_orbits_container {
-    left: 27vw;
+    top: -10vh;
   }
-}
 
-@media (max-width: 750px) {
-  .splash_screen_body {
-    width: 90%;
-  }
```
Comment (ksanmartin909) [inline src/App.css:969]: This breakpoint was pushed down to the bottom by other breakpoints so this looks like  a mess.

Comment (ksanmartin909) [inline src/App.css:969]: Several breakpoints were moved to the bottom or adjusted

I changed the breakpoint for 1170 - 1180 to capture the iPad Air Landscape
I added breakpoint 1040 to target iPadMini - Landscape and Samsung Tab S8
I changed the 500px breakpoint to 1000px to force the iPad Air Portrait (and other similar device widths) to use the vertical mobile device design.
 the 500px breakpoint still exists so all of the other (phone type) devices will still work the same.

File: src/App.css [hunk 3]
```diff
@@ -1523,6 +1640,196 @@ Also, some elements appear to overlap, or be awkwardly placed when viewing alone
...
+
+  .orbit_closeup_back_button_mobile {
+    visibility: visible;
+    position: relative;
+    top: -35.5vh;
+    left: 86vw;
+  }
+
+  .hard_scale_value {
+    transform: scale(1) !important;
```
Comment (rraaschfilament) [inline src/App.css:1681]: What was the reason for this?  I'm assuming it has to override an inline style, but just explain so I know?

Comment (ksanmartin909) [reply src/App.css:1681]: I wrote up an explanation in the PR that actually initiated this which was the ticket where the orbits and earth size get all out of whack at a certain point. 

The problem was that the transform scale calculation for the earths orbit container hits a critical point where things don't line up anymore. 

Let me find the branch where I discuss this further.

Comment (ksanmartin909) [reply src/App.css:1681]: Here is the PR where I added that
https://github.com/rraaschfilament/across-karman/pull/16

Comment (ksanmartin909) [reply src/App.css:1681]: PR-16-description

**Review by rraaschfilament — APPROVED** [review]

**Pull request merged**